dff/core/engine/conditions.py

```diff
@@ -145,13 +147,13 @@ def all_condition_handler(ctx: Context, actor: Actor, *args, **kwargs) -> bool:
 
 @validate_arguments
 def negation(condition: Callable, *args, **kwargs) -> Callable:
```

Comment:
Should we add Callable signature (☝️): `Callable[[Context, Actor, Any, Any], bool]` (AFAIK) here.